x-pack/plugins/lists/public/exceptions/components/builder/exception_item_renderer.tsx

```diff
@@ -152,7 +146,6 @@ export const BuilderExceptionListItemComponent = React.memo<BuilderExceptionList
                           httpService={httpService}
                           indexPattern={indexPattern}
                           listType={listType}
-                          listTypeSpecificIndexPatternFilter={listTypeSpecificIndexPatternFilter}
```

Comment:
I wonder if we really need to remove this since others could use it in the future for other field exceptions.

Reply:
I thought about the same thing - it didn't feel right to leave in unused code, however.

I could add it back, I will wait and see what the Detections Engine team's opinion is

Reply:
I would remove it too if it's unused.